fluent-package/manage-fluent-repositories.sh

```diff
@@ -24,6 +24,7 @@ Example:
   $ $0 upload release-td-agent /tmp/td-agent-release
   $ $0 deb /tmp/td-agent-release 4.2.0
   $ $0 rpm /tmp/td-agent-release 4.2.0
+  $ $0 download-artifacts https://github.com/fluent/fluent-package-builder/pull/587
```

Comment:
The base URL isn't used in actual, so the example might be better to simplify like this:
`$0 download-artifacts 587` or `$0 download-artifacts pull/587`

Reply:
Thanks, It seems reasonable. I'll fix it.